src/renderer/lib/utils.ts

```diff
@@ -1,4 +1,5 @@
 import { characters as charUtils, stages as stageUtils } from "@slippi/slippi-js";
+import path from "path";
```

Comment:
`path` is a node built in which you're importing from the web renderer. you should instead expose required node modules via the context bridge.

Reply:
Didn't know about context bridges, thanks for pointing that out.